code/game/turfs/turf.dm

```diff
@@ -733,3 +733,11 @@ GLOBAL_LIST_EMPTY(created_baseturf_lists)
 		if(turf_to_check.density || LinkBlockedWithAccess(turf_to_check, caller, ID))
 			continue
 		. += turf_to_check
+
+/// Checks if the turf was blessed with holy water OR the area its in is Chapel
+/turf/proc/is_holy()
+	if(locate(/obj/effect/blessing) in src)
+		return TRUE
+	if(istype(loc, /area/chapel))
+		return TRUE
+	return FALSE
```

Comment:
Should probably replace all instances that check for holy precence with this proc. For example, EMPs and explosions sourced from unholy energies can be blocked by having a blessing on the ground.

Reply:
That'll make the chapel immune to said EMPs and explosions immune to it too, is that okay?

Reply:
Yes, that's fine. They are holy locations